ScienceJournal/WrappedModels/TrialStats.swift

```diff
@@ -38,7 +38,7 @@ class TrialStats {
       return proto.hasStatStatus ? proto.statStatus : .needsUpdate
     }
     set {
-      proto.statStatus = status
+      proto.statStatus = newValue
```

Comment:
This one is kinda surprising to me, we were never updating the protos' statStatus 😮

Reply:
we set the value (or at least we had been attempting to, unsuccessfully) but never read it. If I recall android had never implemented reading the value so we decided to do the same. But glad the linter can catch these types of errors.